lib/streamlit/elements/time_widgets.py

```diff
@@ -544,12 +563,21 @@ def _date_input(
             date_input_proto.help = dedent(help)
 
         date_input_proto.label = label
-        date_input_proto.default[:] = [
-            date.strftime(v, "%Y/%m/%d") for v in parsed_values.value
-        ]
 
-        date_input_proto.min = date.strftime(parsed_values.min, "%Y/%m/%d")
-        date_input_proto.max = date.strftime(parsed_values.max, "%Y/%m/%d")
+        clearable = parsed_values.value is None
+        if clearable:
+            date_input_proto.default[:] = []
+        elif isinstance(parsed_values.value, Iterable) or isinstance(
```

Comment:
Every `Sequence` is an iterable. So, we can remove this `isinstance(parsed_values.value, Sequence):` check here

Reply:
Good to know, it's updated now 👍